infrastructure/kube/keep-test/monitoring/configmap.yaml

```
@@ -0,0 +1,656 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: monitoring
+  name: prometheus-conf
```

Do we need `-conf` suffix?
```suggestion
  name: prometheus
```